cpu_scheduling_algorithms/non_preemptive_sjf_scheduling.cpp

```diff
@@ -0,0 +1,312 @@
+/**
+ * @file
+ * @brief Implementation of SJF CPU scheduling algorithm
+ * @details
+ * shortest job first (SJF), also known as shortest job next (SJN), is a
+ * scheduling policy that selects for execution the waiting process with the
+ * smallest execution time. SJN is a non-preemptive algorithm. Shortest
+ * remaining time is a preemptive variant of SJN.
+ * @link https://www.guru99.com/shortest-job-first-sjf-scheduling.html
```

Comment:
Use an <a> tag this link doesn't work like this

Reply:
Done